e2e/fcm_e2e_test.go

```diff
@@ -71,13 +71,17 @@ func (s *FcmE2ETestSuite) setupFcmPusher(appName string) (*firebaseMock.MockPush
 	statsReport, err := extensions.NewStatsD(s.vConfig, logger, statsdClientMock)
 	s.Require().NoError(err)
 
+	limit := s.vConfig.GetInt("gcm.rateLimit.rpm")
```

Comment:
what happens if the property is nil? is there a default value?

Reply:
we are using default.yaml config with a value of 20